apps/arweave/include/ar_config.hrl

```diff
@@ -186,6 +184,8 @@
 	defragmentation_modules = [],
 	block_throttle_by_ip_interval = ?DEFAULT_BLOCK_THROTTLE_BY_IP_INTERVAL_MS,
 	block_throttle_by_solution_interval = ?DEFAULT_BLOCK_THROTTLE_BY_SOLUTION_INTERVAL_MS,
+	tls_cert_file = not_set, %% required to enable TLS
+  tls_key_file = not_set,  %% required to enable TLS
```

Comment:
Even spacing suggested

Reply:
not practicing what I'm preaching, fixed